packages/services/api/src/modules/audit-logs/module.graphql.ts

```diff
@@ -0,0 +1,400 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  """
+  AuditLog is a record of actions performed by users in
+  the organization. It is used to track changes and
+  actions performed by users.
+  """
+  interface AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    record: AuditLogIdRecord!
+  }
+
+  type AuditLogIdRecord {
+    userId: ID!
+    userEmail: String!
+    organizationId: ID!
+    """
+    User can be null if the user is deleted
+    """
+    user: User
+    """
+    Organization can be null if the organization is deleted
+    """
+    organization: Organization
+  }
+
+  type AuditLogConnection {
+    nodes: [AuditLog!]!
+    total: Int!
+  }
+
+  """
+  AuditLogFilter is used to filter audit logs by
+  date range and user id.
+  """
+  input AuditLogFilter {
+    startDate: DateTime
+    endDate: DateTime
+    userId: ID
+  }
+
+  """
+  AuditLogPaginationFilter is used to paginate audit logs.
+  By default, it returns 25 records with an offset of 0.
+  """
+  input AuditLogPaginationFilter {
+    limit: Int = 25
+    offset: Int = 0
+  }
+
+  extend type Query {
```

Comment:
I think @n1ru4l wanted this to be defined as `extend type Organization`, and then you don't need a selector argument